frontend/src/APIClients/CamperAPIClient.ts

```diff
@@ -2,9 +2,40 @@ import { BEARER_TOKEN } from "../constants/AuthConstants";
 import {
   WaitlistedCamper,
   UpdateWaitlistedStatusType,
+  EditCamperInfoFields,
+  Camper,
 } from "../types/CamperTypes";
 import baseAPIClient from "./BaseAPIClient";
 
+const updateCampersById = async (
+  id: Array<string>,
+  camperData: EditCamperInfoFields,
+): Promise<Array<Camper>> => {
+  try {
+    const body = { camperIds: id, ...camperData };
+
+    // Can't send Map<string, string> over HTTP so need to convert to Object before
+    if (camperData.formResponses) {
+      body.formResponses = Object.assign(
+        {},
+        ...Array.from(camperData.formResponses.entries()).map(
+          ([question, response]) => ({
+            [question]: response,
+          }),
+        ),
```

Comment:
nit: is this different from `new JSONObject(camperData.formResponses);`?

Reply:
I'm not sure which function you are referring to. TS didn't recognize it and I can't seem to find it online. Could you link it for me?

Reply:
I tried that with the following code:
```
if (camperData.formResponses instanceof Map) {
      // body.formResponses = Object.assign(
      //   {},
      //   ...Array.from(camperData.formResponses.entries()).map(
      //     ([question, response]) => ({
      //       [question]: response,
      //     }),
      //   ),
      // );

      body.formResponses = Object.fromEntries(camperData.formResponses);
    }
```
But I get the following error:
```
(property) formResponses?: Map<string, string> | undefined
Type '{ [k: string]: string; }' is missing the following properties from type 'Map<string, string>': clear, delete, forEach, get, and 8 more.ts(2740)
```